pyvista/core/pointset.py

```diff
@@ -919,15 +919,26 @@ def n_lines(self) -> int:
     def n_verts(self) -> int:
         """Return the number of vertices.
 
+        A vertex is a single element cell referencing a single point.
+        See `pyvista.PolyData.n_points` for the more common measure.
+
         Examples
         --------
         Create a simple mesh containing just two points and return the
-        number of vertices.
+        number of vertices. By default, when constructing a PolyData with points but no cells,
+        vertices are automatically created, one per point.
 
         >>> import pyvista
         >>> mesh = pyvista.PolyData([[1.0, 0.0, 0.0], [1.0, 1.0, 1.0]])
-        >>> mesh.n_verts
-        2
+        >>> mesh.n_points, mesh.n_verts
+        (2, 2)
+
+        If any other cells are specifed, these vertices are not created.
```

Comment:
Not exactly. It's perfectly okay to have points that are registered to face cells and as their own vertex cell. For example:

```py
import pyvista as pv
from pyvista import examples

mesh = examples.download_bunny_coarse()
mesh.verts = mesh._make_vertex_cells(mesh.n_points)
```
```py
>>> mesh.n_verts
872
>>> mesh.plot()
```
![screenshot (28)](https://github.com/pyvista/pyvista/assets/22067021/e3123974-7d1a-4b3a-8bb6-278adb3d9fad)

There used to be some example datasets with this scenario but I can't remember which

Reply:
I think that what is being conveyed is pyvista's model when instantiating a `pyvista.PolyData` object, so this could probably just be clarified a bit that this example is chosen to demonstrate the difference.

Reply:
Ah yes, that is the behavior when using the constructor

Reply:
Yup, was trying to clarify the behavior of the PolyData constructor in that specific example, where vertices are constructed only implicitly

Reply:
In that case, great as is! Sorry for my confusion -- trying to blast through lots of notifications

Reply:
Side note: so many times I wanted to plot points as well as faces for a mesh in a single call (because I'm lazy). TIL I need `verts` for that.